neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/facade/impl/Neo4jTransactionFacade.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.neo4j.remote.facade.impl;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.hibernate.ogm.datastore.neo4j.remote.json.impl.Statements;
+
+/**
+ * REST API to manage a transaction.
+ *
+ * @author Davide D'Alto
+ */
+@Path("/")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface Neo4jTransactionFacade {
+
+	@POST
+	@Path("/transaction")
+	Response beginTransaction();
+
+	@POST
+	@Path("/transaction/{transactionId}/commit")
+	Response commit(@PathParam("transactionId") long transactionId);
+
+	@DELETE
+	@Path("/transaction/{transactionId}")
+	Response rollback(@PathParam("transactionId") long transactionId);
+
+	@POST
+	@Path("/transaction/commit")
+	Response executeQuery(Statements statements);
```

Comment:
This is the Ne4j Rest transaction API, if you want to execute some statements in one request inside a new transaction, you need to call the the /transaction/commit and pass the queries.

The path looks strange if you compare it to the method name, but the method name is telling you what's happening.

Maybe I will change the name to something like  `executeQueriesInNewTransaction`